misc/acrn-config/library/board_cfg_lib.py

```diff
@@ -34,7 +34,7 @@
 TSN_DEVS = ["8086:4b30", "8086:4b31", "8086:4b32", "8086:4ba0", "8086:4ba1", "8086:4ba2",
             "8086:4bb0", "8086:4bb1", "8086:4bb2", "8086:a0ac", "8086:43ac", "8086:43a2"]
 GPIO_DEVS = ["8086:4b88", "8086:4b89"]
-TPM_PASSTHRU_BOARD = ['whl-ipc-i5', 'whl-ipc-i7', 'tgl-rvp']
+TPM_PASSTHRU_BOARD = ['whl-ipc-i5', 'whl-ipc-i7', 'tgl-rvp', 'ehl-crb-b']
```

Comment:
Does this mean that if someone wants to create their own board with the `acrn-config`, and wants to expose the TPM2 device on it (as a pass-through device), they **must** also edit this file to make this happen?

Reply:
Currently we officially supports these boards for TPM passthru and hard coded them here.  If users want their own board configured with TPM passthru to the pre-launched VM, they should also edit this file and configure vm->mmio_resources->TPM2 = y in scenario xml.

Reply:
@NanlinXie , this is an example of something we should avoid in our ACRN configuration tool.